_rules/focusable-element-has-visible-focus-f4e323.md

```diff
@@ -0,0 +1,390 @@
+---
+id: f4e323
+name: Focusable element has visible focus
+rule_type: atomic
+description: |
+  This rule checks that each focusable element has visible focus indication
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.4.7: # Focus Visible (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: satisfied
+input_aspects:
+  - CSS styling
+  - DOM tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+The rule applies to any [focusable][] element in a document with several [focusable][] elements when the element is part of [sequential focus navigation][].
+
+## Expectation
+
+For each target element, there exists a set of [focus indicators][focus indicator] which are not all also [focus indicators][focus indicator] for another [focusable][] element (the same for all the indicators).
```

Comment:
```suggestion
For each target element, there exists a set of [focus indicators][focus indicator] which are not all also [focus indicators][focus indicator] for another [focusable][] element.
```

It seemed a little detached 🤔

Reply:
The precision is needed essentially for cases like Passed Example 7 where there are several focusable elements in a row and focus is indicated by lighting a dot on both side of each focusable. Then, each focus indicator is a focus indicator for 2 different elements. So for elements in the middle, all of their focus indicators are also focus indicator for another element, however this is not the same other element for all focus indicator, so this is good.